en/orm/saving-data.rst

```diff
@@ -1030,6 +1030,33 @@ from request data your POST data should look like::
         'associated' => ['Courses._joinData']
     ]);
 
+In some situations, even the junction table can have its own associations. For example if Courses hasMany Assignments, we could have a data like this::
```

Comment:
`Courses` wouldn't be the join table model here, it would by default be `CoursesStudents`. Considering the [**examples for the `through` option**](https://github.com/cakephp/docs/blob/3437abd8f08eccf6e0aa57f4bb30bff69ab6b791/en/orm/associations.rst#using-the-through-option), maybe use `CourseMemberships` here too?

Reply:
@ndm2 but I just repeated (and extended) the last example, so what do you suggest for the data array?

Reply:
The data array is OK I think (not sure about the duplicate `grade` though), what I ment was that `Courses` is the target table of the `belongsToMany` association, it's not the join table, by default, if not configured via the `through` option, the join table would be `CoursesStudents`, just like @markstory noted, ie your example would need to say:

>  For example if `CoursesStudents hasMany Assignments`, ...

My suggestion would be to maybe incorporate the `through` option here and use a name like `CourseMemberships` for the join table. So it should just be a matter of describing that in the text, something along the lines of:

> For example if `Students belongsToMany Courses` [through](https://github.com/cakephp/docs/blob/3437abd8f08eccf6e0aa57f4bb30bff69ab6b791/en/orm/associations.rst#using-the-through-option) `CourseMemberships`, and `CourseMemberships hasMany Assignments`, ...